BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -103,6 +100,11 @@ - (instancetype)initWithSDKClient:(BOXContentClient *)SDKClient
         _connectionData = [[NSMutableData alloc] init];
         _hostsThatCanUseWebViewDirectly = [NSMutableSet set];
 
+        NSURLSessionConfiguration *sessionConfiguration = [NSURLSessionConfiguration ephemeralSessionConfiguration];
+        _URLSession = [NSURLSession sessionWithConfiguration:sessionConfiguration
+                                                    delegate:self
+                                               delegateQueue:nil];
```

Comment:
minor: create a delegateQueue ourselves to have control over maxConcurrentOperationCount to 1.